pyfvtool/cell.py

```diff
@@ -311,6 +316,7 @@ def cellLocations(m: MeshStructure):
         z = np.zeros((1,1,N[2]))
         z[0, 0, :] = m.cellcenters.z
         Z = createCellVariable(m, np.tile(z, (N[0], N[1], 1)))
+        return X, Y, Z # added by MW 240118, double check if OK
```

Comment:
Please check. This return statement seemed to be missing

Reply:
Looks fine!